diff --git a/bpf/common/ringbuf.h b/bpf/common/ringbuf.h
index 7baf44b5..b59f69b7 100644
--- a/bpf/common/ringbuf.h
+++ b/bpf/common/ringbuf.h
@@ -18,7 +18,7 @@
 // we can share them later if we find is worth not including code per duplicate
 struct {
     __uint(type, BPF_MAP_TYPE_RINGBUF);
-    __uint(max_entries, 1 << 20);
+    __uint(max_entries, 1 << 22);
     __uint(pinning, OBI_PIN_INTERNAL);
 } events SEC(".maps");
 
